taxi_service/urls.py

```diff
@@ -13,9 +13,15 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+
+
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+
+import taxi.views
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("taxi/", include(("taxi.urls", "taxi"), namespace="taxi")),
```

Comment:
in this task you only need to specify path for index page so change the url here and remove the path below